src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -71,6 +78,25 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
     const [toolbarState, setToolbarState] = useState<'show' | 'hide'>('show');
     const [toolbarHeight, setToolbarHeight] = useState(0);
     const [notificationsHeight, setNotificationsHeight] = useState(0);
+    const contentRef = useRef<HTMLDivElement>(null);
+
+    useResizeObserver(
+      contentRef,
+      throttle(entry => {
```

Comment:
Resize observations are already throttled natively. Did you have any issues here?

Reply:
it's not present here anymore